### DISCUSSION TITLE
URGENT: Official WebSite is down several times

### DISCUSSION THREAD

**Discussion by shakesBeardZ** [fastapi/fastapi#8851] [Questions]
the official website is down several times today.
<img width="400" src="https://user-images.githubusercontent.com/15571628/117094468-8850c500-ad6c-11eb-9f86-a2ff5220fcca.png" />

**Comment by Mause** [comment]
Seems fine to me? Are you sure this isn't an issue with your connection?

**Comment by FlorianLudwig** [comment]
@shakesBeardZ This looks like an issue on your side / your service provider / dns provider.

please try a setting a manaul DNS server, for example `8.8.8.8` (google) or `1.1.1.1` (cloudflare) and see if the issue persists.

**Comment by Joshuah143** [comment]
I also cant reproduce. Worth closing the issue?
